modules/extloads/src/ExtLoadsDX_Registry.txt

```diff
@@ -0,0 +1,44 @@
+# Registry for ExternalLoadsDX in the FAST Modularization Framework
+# This Registry file is used to create ExtLoadsDX_Types which contains data used in the ExtLoads module for data exchange with external drivers.
+# It also contains copy, destroy, pack, and unpack routines associated with each defined data types.
+# See the NWTC Programmer's Handbook for further information on the format/contents of this file.
+#
+# Entries are of the form
+# <keyword>	<ModuleName/ModName>	<TypeName>	<FieldType>	<FieldName>	<Dims>	<IO>	<DNAME>	<DESCRIP>	<UNITS>
+#
+# Use ^ as a shortcut for the value in the same column from the previous line.
+###################################################################################################################################
+# File last committed $Date$
+# (File) Revision #: $Rev$
+# URL: $HeadURL$
+###################################################################################################################################
+# ...... Include files (definitions from NWTC Library) ............................................................................
+include	Registry_NWTC_Library.txt
+
+# ..... Inputs ....................................................................................................................
+# Define inputs that are contained on the mesh here:
+typedef ExtLoadsDX/ExtLdDX       InputType      R8Ki            twrDef        {:}     -       -       "Deformations on the tower  - to send to external driver"
+typedef ^       InputType      R8Ki            bldDef        {:}     -       -       "Deformations on all blades - to send to external driver"
+typedef ^       InputType      R8Ki            hubDef        {:}     -       -       "Deformations on the hub  - to send to external driver"
+typedef ^       InputType      R8Ki            nacDef        {:}     -       -       "Deformations the nacelle - to send to external driver"
+typedef ^       InputType      R8Ki            bldRootDef    {:}     -       -       "Deformations of the blade root nodes - to send to external driver"
+typedef ^       InputType      R8Ki            twrRefPos     {:}     -       -       "Reference position of the tower nodes - to send to external driver"
+typedef ^       InputType      R8Ki            bldRefPos     {:}     -       -       "Reference position of the all blade nodes - to send to external driver"
+typedef ^       InputType      R8Ki            hubRefPos     {:}     -       -       "Reference position of the tower nodes - to send to external driver"
+typedef ^       InputType      R8Ki            nacRefPos     {:}     -       -       "Reference position of the all blade nodes - to send to external driver"
+typedef ^       InputType      R8Ki            bldRootRefPos {:}     -       -       "Reference position of the blade root nodes - to send to external driver"
+typedef ^       InputType      IntKi           nBlades       {:}     -       -       "Number of blades"
```

Comment:
These integers (and possibly some other R8Ki variables below) seem like they should be parameters of the module, not inputs.

Reply:
I agree!

@gantech, let's figure out how to resolve this.

Reply:
Maybe a better solution: PR #2001